src/external/steamwebrtc/CMakeLists.txt

```diff
@@ -686,6 +692,18 @@ if(WIN32)
 		Secur32
 		Iphlpapi
 		)
+elseif(CMAKE_SYSTEM_NAME MATCHES Darwin)
```

Comment:
Thanks for the suggestion, but it doesn't work. If `Darwin` is changed to `IOS` the mac build fails because `WEBRTC_MAC` is not defined which leads to compilation errors.

Moreover, using `Darwin` is more cohesive as it is how the [main CMakeLists.txt:173](https://github.com/ValveSoftware/GameNetworkingSockets/blob/d5f855967440eeb5b4d5798bebe179ef868ac6af/CMakeLists.txt#L162) tests for macos.

Reply:
You edited your previous comment from `IOS` to `APPLE`. Yes, if line 695 is changed from:

`elseif(CMAKE_SYSTEM_NAME MATCHES Darwin)`

to:

`elseif(APPLE)`

It does indeed compile successfully. However, this breaks the cohesion with the main [CMakeLists.txt:173](https://github.com/ValveSoftware/GameNetworkingSockets/blob/d5f855967440eeb5b4d5798bebe179ef868ac6af/CMakeLists.txt#L162). Might be better to submit a separate pull request for this.

Reply:
Thanks, Jake. You make a good point. I think it just needs to be a separate pull request to maintain cohesion with the primary `CMakeLists.txt` and to focus on one purpose at a time. The purpose of this pull request is to solve compilation errors on macos when enabling `steamwebrtc`.

Cheers and wishing you all the best.